presidio-analyzer/Dockerfile.local.conda

```diff
@@ -0,0 +1,54 @@
+FROM python:3.7-slim
```

Comment:
Is this docker file used for development? How would a new user use it? Why not use Pipenv locally?

Reply:
I can also remove this file (same reasoning as `requirements.txt`).

But another reason that I created this file is that it's a lot faster to iterate when testing.  Your current `Dockerfile.local` reinstalls all the requirements any time there is any change to the code.  This takes relatively long especially when making changes that don't affect any of the requirements.  So I added a step that installs from the `Pipfile.lock` (via jq and requirements.txt, super hacky to use conda) and then installs models, then adds the code to the docker.  The important part is adding the code after installing the requirements and models, so the diffs between the images are super small.

Lastly, I should mention that in quarantine, I am basically running internet through my cell phone, so it's slow and I have a limited amount of data.  So I'm much more conscious of large downloads than I normally would be.  This is probably a big reason why I am so against making the `en_core_web_lg` model a requirement in the `Pipfile`.

Reply:
One more point, I found that having the `Dockerfile.python.deps` in the base folder and `Dockerfile` and `Dockerfile` in `presidio-analyzer` confusing.  Since there is only one python service in this library, I wanted to create a single Dockerfile that I could rebuild to test my changes within the service.  In the long term, you might want to consider making each service a separate repo and then using git submodules.  The python analyzer in particular seems like a standalone product that can be useful outside of the service as a whole.

Reply:
On your second point, I agree it's confusing to have it in the root of the repo, but the reason it's divided into two is to answer your first point. you have a base docker image which you install only once, and then you use the analyzer Dockerfile to build your code only, without having to reinstall dependencies.

Reply:
I deleted this file.